optimum/intel/openvino/quantization.py

```diff
@@ -137,11 +141,145 @@ def quantize(
         >>> optimized_model = OVModelForSequenceClassification.from_pretrained("./quantized_model")
         ```
         """
+        if isinstance(self.model, OVBaseDecoderModel) and self.model.use_cache:
+            self._quantize_ovcausallm(
+                calibration_dataset,
+                save_directory,
+                batch_size,
+                data_collator,
+                remove_unused_columns,
+                **kwargs,
+            )
+        elif isinstance(self.model, OVBaseModel):
+            self._quantize_ovbasemodel(
+                calibration_dataset,
+                save_directory,
+                batch_size,
+                data_collator,
+                remove_unused_columns,
+                **kwargs,
+            )
+        elif isinstance(self.model, torch.nn.Module):
+            self._quantize_torchmodel(
+                calibration_dataset,
+                save_directory,
+                quantization_config,
+                file_name,
+                batch_size,
+                data_collator,
+                remove_unused_columns,
+            )
+        else:
+            raise TypeError(f"Unsupported model type: {type(self.model)}")
+
+    def _quantize_ovbasemodel(
+        self,
+        calibration_dataset: Dataset,
+        save_directory: Union[str, Path],
+        batch_size: int = 1,
+        data_collator: Optional[DataCollator] = None,
+        remove_unused_columns: bool = True,
+        **kwargs,
+    ):
+        save_directory = Path(save_directory)
+        save_directory.mkdir(parents=True, exist_ok=True)
+
+        calibration_dataloader = self._get_calibration_dataloader(
+            calibration_dataset=calibration_dataset,
+            batch_size=batch_size,
+            remove_unused_columns=remove_unused_columns,
+            data_collator=data_collator,
+        )
+
+        quantization_dataset = nncf.Dataset(calibration_dataloader, lambda x: x)
+        quantized_model = nncf.quantize(
+            self.model.model,
+            quantization_dataset,
+            model_type=nncf.ModelType.TRANSFORMER if not kwargs.get("model_type") else kwargs.get("model_type"),
+            fast_bias_correction=True
+            if not kwargs.get("fast_bias_correction")
+            else kwargs.get("fast_bias_correction"),
```

Comment:
it looks equivalent to:
```suggestion
            fast_bias_correction=kwargs.get("fast_bias_correction", True),
```